Dockerfile

```diff
@@ -0,0 +1,44 @@
+# Stage 1: Install Dependencies
```

Comment:
Why new Docker configuration? I think we can use existing Dockerfile.dev for it?